### PR TITLE
Fix building universal apk with current nightly

### PR DESCRIPTION
**Pull request by Dobby233Liu**
This fix is verified to work with a real project, but I haven't tried to create a bundle yet, but it should still work. Sorry for the 15 commits.

### CODE REVIEW

File: rapt/prototype/gradle/wrapper/gradle-wrapper.properties [hunk 1]
```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-6.1.1-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.9.1-bin.zip
```
Comment (Dobby233Liu) [inline rapt/prototype/gradle/wrapper/gradle-wrapper.properties:3]: should I change it back to 6.1.1? (sorry for reposting)

File: rapt/templates/settings.gradle [hunk 2]
```diff
@@ -3,3 +3,4 @@ include ':renpyandroid', ':app'
 {% else %}
 include ':renpyandroid', ':app', ':ff1', ':ff2', ':ff3', ':ff4'
 {% endif %}
+rootProject.name="app"
```
Comment (Dobby233Liu) [inline rapt/templates/settings.gradle:6]: i don't know but this may be unnecessary

**Comment by renpytom** [comment]
Sorry, what problems is this trying to resolve?

**Comment by renpytom** [comment]
Thank you, woke up and figured this out.

I merged this, though next time, it might make sense to squash this into a single commit. I also want to stick with gradle 6.1.1 for now, as it works and is what my Android Studio is using for new projects.

**Pull request merged**